### DISCUSSION TITLE
ExportAction::make()->modifyQueryUsing() does not appear to exist

### DISCUSSION THREAD

**Discussion by HolyProfanity** [filamentphp/filament#11434] [Help] [answered]
### Package

Actions

### Package Version

V3.2

### How can we help you?

According to documentation (https://filamentphp.com/docs/3.x/actions/prebuilt-actions/export#modifying-the-export-query), the method "ExportAction::make()->modifyQueryUsing()" shipped with Filament 3.2 should exist for ExportAction class but I can't seen to find it anywhere. Has anyone suceeded in calling the new method?
![Captura de tela 2024-02-15 112106](https://github.com/filamentphp/filament/assets/57575002/39b381ec-244c-4179-8cd7-902f8f503359)

**Comment by leandrocfe** [comment, marked as answer]
What is the Filament version 3.x.x?
Did you update it with `composer update -W`?

**Reply by HolyProfanity** [reply]
The version is 3.2.34. Running composer update -W solved the problem. Thanks a bunch!
